__tests__/news.test.js

```diff
@@ -22,11 +23,74 @@ describe('/api/topics', () => {
             return request(app).get('/api/topics')
             .expect(200)
             .then(({body}) => {
+                expect(body.length).not.toBe(0);
                 body.forEach((topic) => {
                 expect(topic).toHaveProperty('description')
                 expect(topic).toHaveProperty('slug')
             });
             });
         });
-    })
-})
+    });
+});
+
+describe('/api/articles', () => {
+    describe('GET articles', () => {
+        test('Returns status 200 if the request is successful', () => {
+            return request(app).get('/api/articles')
+            .expect(200);
+        });
+        test('Returns an array of article objects sorted in descending order by date', () => {
+            return request(app).get('/api/articles')
+            .expect(200)
+            .then(({body}) => {
+                expect(body).toBeSortedBy('created_at', {descending : true})
+            })
+        });
+        test('Returns an array of article objects including a comment count', () => {
+            return request(app).get('/api/articles')
+            .expect(200)
+            .then(({body}) => {
+                expect(body.length).not.toBe(0)
+                body.forEach((article) => {
+                    expect(article).toHaveProperty('author');
```

Comment:
You might want to have a look at the jest matcher: `.objectContaining` here. but this is a matter of driver's choice